### ISSUE TITLE
Difficulty getting started in Windows

### ISSUE THREAD

**Issue by mentoc3000** [timbod7/haskell-chart#206]
I have spent several hours trying to get the basic Chart examples to compile and run in a Windows environment.  I have tried to install Cairo and the associated Haskell packages but have found no success.  A step by step tutorial for getting set up on Windows would be tremendously helpful and would lower barriers to entry to use this package and Haskell in general.

**Comment by bravit** [comment]
Thanks for reporting. You are right, we have to do something about it.

**Comment by mpilgrem** [comment]
I am a Windows 10 user and I have been able to get the Chart examples to work with Cairo. I have added my experiences to the 'Getting started' page of the Chart wiki. In that respect:

1. Out-of-date `zlib1.dll` files on the path caused me the most difficulty, until I realised that was the problem; and

2. The list of packages to be added via MSYS2 is based on [Using haskell-gi in Windows](https://github.com/haskell-gi/haskell-gi/wiki/Using-haskell-gi-in-Windows). However, MSYS2 has recently [dropped](https://github.com/msys2/MINGW-packages/issues/5299) `webkitgtk3`. I think it is not needed for Chart but cannot test that as I already have it installed in my own set up.
